pkg/datapath/loader/datapath_sha256.go

```diff
@@ -0,0 +1,5 @@
+package loader
```

Comment:
In case we decide to add it to the tree, let's add a comment header saying:

```go
// Code generated by $(cmd). DO NOT EDIT
```

So it will be recognized by tooling and e.g., GitHub as auto-generated. Also see https://pkg.go.dev/cmd/go#hdr-Generate_Go_files_by_processing_source:

Reply:
For compatibility with the standard Go tooling we need to commit generated files to the tree. The `go` tool does not run any extra build steps by default, so if we don't commit the generated file, anyone checking out the code and running `go build` or `go test` will get an error about `DatapathSHA256` being undefined.

I've added the "Code generated..." header.